webpack.config.js

```diff
@@ -0,0 +1,22 @@
+var path = require('path');
+var SRC_DIR = path.join(__dirname, '/client/src');
+var DIST_DIR = path.join(__dirname, '/client/dist');
+
+module.exports = {
+  entry: `${SRC_DIR}/app.jsx`,
+  output: {
+    filename: 'bundle.js',
+    path: DIST_DIR
+  },
+  module: {
+    rules: [
+      {
+        test: /\.jsx?/,
```

Comment:
If this doesn't work, I'd try "/\.jsx?$/". But it's cool if it does work!

Reply:
Seems to be working!